ci-scripts/linux/local-build/linux_build.sh

```diff
@@ -0,0 +1,559 @@
+		[de_CLONING_ERROR_RETRY]="Fehler beim Klonen. Bitte versuche es erneut."
+		[fr_CLONING_ERROR_RETRY]="Erreur de clonage. Veuillez réessayer."
+		[gl_CLONING_ERROR_RETRY]="Erro na clonación. Inténtao de novo."
+		[pt_CLONING_ERROR_RETRY]="Erro na clonagem. Tente novamente."
+		
+		[es_INVALID_OPTION_RETRY]="Opción no válida. Intente de nuevo."
+		[en_INVALID_OPTION_RETRY]="Invalid option. Please try again."
+		[it_INVALID_OPTION_RETRY]="Opzione non valida. Riprova."
+		[de_INVALID_OPTION_RETRY]="Ungültige Option. Bitte versuche es erneut."
+		[fr_INVALID_OPTION_RETRY]="Option invalide. Veuillez réessayer."
+		[gl_INVALID_OPTION_RETRY]="Opción non válida. Inténtao de novo."
+		[pt_INVALID_OPTION_RETRY]="Opção inválida. Tente novamente."
+  		
+    		[en_SPECIFY_DIRECTORY]="Specify the path"
+		[it_SPECIFY_DIRECTORY]="Specificare il percorso"
+		[de_SPECIFY_DIRECTORY]="Pfad angeben"
+		[fr_SPECIFY_DIRECTORY]="Spécifier le chemin"
+		[gl_SPECIFY_DIRECTORY]="Especificar a ruta"
+		[pt_SPECIFY_DIRECTORY]="Especificar o path"
+
+		[en_INTRO_PATH]="Enter the path-> "
+		[it_INTRO_PATH]="Inserisci il percorso-> "
+		[de_INTRO_PATH]="Geben Sie den Pfad ein-> "
+		[fr_INTRO_PATH]="Entrez le chemin-> "
+		[gl_INTRO_PATH]="Introduce o path-> "
+		[pt_INTRO_PATH]="Introduza o path-> "		
+  
+  		[es_TAHOMA2D_FOLDER_FOUND_AT]='Carpeta "tahoma2d" encontrada en: '
+		[en_TAHOMA2D_FOLDER_FOUND_AT]="Folder 'tahoma2d' found at: "
+		[it_TAHOMA2D_FOLDER_FOUND_AT]="Cartella 'tahoma2d' trovata in: "
+		[de_TAHOMA2D_FOLDER_FOUND_AT]="Ordner 'tahoma2d' gefunden unter: "
+		[fr_TAHOMA2D_FOLDER_FOUND_AT]="Dossier 'tahoma2d' trouvé à : "
+		[gl_TAHOMA2D_FOLDER_FOUND_AT]="Cartafol 'tahoma2d' atopado en: "
+		[pt_TAHOMA2D_FOLDER_FOUND_AT]="Pasta 'tahoma2d' encontrada em: "		
+				
+		[es_DIRECTORY_NOT_FOUND_RETRY]='Directorio no encontrado. Reintentar. '
+		[en_DIRECTORY_NOT_FOUND_RETRY]="Directory not found. Retry. "
+		[it_DIRECTORY_NOT_FOUND_RETRY]="Directory non trovato. Riprova. "
+		[de_DIRECTORY_NOT_FOUND_RETRY]="Verzeichnis nicht gefunden. Erneut versuchen. "
+		[fr_DIRECTORY_NOT_FOUND_RETRY]="Répertoire introuvable. Réessayer. "
+		[gl_DIRECTORY_NOT_FOUND_RETRY]="Directorio non atopado. Reintentar. "
+		[pt_DIRECTORY_NOT_FOUND_RETRY]="Diretório não encontrado. Tente novamente. "
+	)
+
+    local idioma=$(echo $LANG | cut -d_ -f1)
+    local mensaje=${msg[${idioma}_$clave]:-${msg[en_$clave]}}
+
+    printf "%s" "$mensaje"
+}
+
+# ======================================================================
+# Functions
+# ======================================================================
+
+function _checkroot() {
+    # Check if the script is running WITH root privileges (and exit if true)
+    if [ "$EUID" -eq 0 ]; then
+        echo ''
+        echo "$(_msg ROOTCHECK)"
+        echo ''
+        exit 1
+    fi
+}
+
+function _checkdepends() {
+	# Check if dependencies have been installed
+	if [ ! -e "$SCRIPT_DIR/ok-packages" ]; then
+	echo""
```

Comment:
Some `if` blocks are indented and some are not. They should probably all be indented for readability.